templates/around/map.html

```diff
@@ -11,6 +11,9 @@
 
 {% block main %}
   <h1>Find someone or something close to <em>you</em></h1>
+  <button type="button" onclick="userCenter()">
+    <img height="100%" src="/static/images/recenter.png"/>
```

Comment:
remove the quotes and the ending slash ;)
(to be consistent with the writing of the code in Neomad)